ts/core/MmlTree/MmlNode.ts

```diff
@@ -330,6 +347,36 @@ export abstract class AbstractMmlNode extends AbstractNode implements MmlNode {
     this.attributes.setList(attributes);
   }
 
+  /**
+   * @override
+   *
+   * @param {boolean} keepIds   True to copy id attributes, false to skip them.
+   */
+  public copy(keepIds: boolean = false): AbstractMmlNode {
```

Comment:
I am surprised typescript allows this, as it does not faithfully implement the interface. That could be the source of a future compile error.
Maybe add a comment?